deploy/helm/flightctl/templates/flightctl-api-networkpolicy.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: flightctl-api-from-ingress
+  namespace:  {{ .Release.Namespace }}
+spec:
+  ingress:
+  - from:
+    - namespaceSelector:
+        matchLabels:
+          network.openshift.io/policy-group: ingress
```

Comment:
ok I suspect this fails because somehow kind in github CI works with network policies, and we are not allowing access from the NodePort/local ip